src/uwtools/drivers/forecast.py

```diff
@@ -72,12 +74,23 @@ def create_model_config(self, config_obj, outconfig_file, base_file=None):
         msg = f"Config file {outconfig_file} created"
         logging.info(msg)
 
-    def stage_fix_files(self):
+    def stage_fix_files(self, run_directory, config_obj):
 
         ''' Holds the knowledge for how to modify a list of fix files and
-        stages them in the working directory. Likely gets all its info from
-        config_obj. Calls data mover tool (could be python copy). Fix files
-        usually are specific to a given named grid and resolution. '''
+        stages them in the working directory. Calls data mover tool (could
+        be python copy). Fix files usually are specific to a given named
+        grid and resolution. '''
+
+        files_to_stage = config_obj.get('static', {})
+        for target, file_path in files_to_stage.items():
+            if os.path.isfile(file_path):
+                file = os.path.join(run_directory, target)
+                shutil.copyfile(file_path, file)
+                msg = f"File {file_path} staged in working directory at {target}"
```

Comment:
Here, too, is it a "working" directory, or a "run" directory. IMO we should use consistent terminology.